subworkflows/nf-core/fastq_find_mirna_mirdeep2/tests/main.nf.test.snap

```diff
@@ -0,0 +1,17 @@
+{
+    "smrnaseq - fasta - single_end": {
+        "content": [
+            [
+                "versions.yml:md5,10138b74aed5b2658c26ddf80ff391d5",
+                "versions.yml:md5,631c0428c28d5355f0e3e9bd790bd77d",
```

Comment:
the workflow is emitting only version outputs ? no other outputs ?

Reply:
captured the unstable outputs in the snapshot now :)